{"name": "test_login_with_valid_credentials[aziguy_two@gmail.com-secure_password_two]", "status": "passed", "parameters": [{"name": "email_address", "value": "'aziguy_two@gmail.com'"}, {"name": "password", "value": "'secure_password_two'"}], "start": 1722274043814, "stop": 1722274045463, "uuid": "c9059143-917c-450e-bb04-013a0fe91298", "historyId": "d3b625c07d76a7cc67f9ba15f9536bef", "testCaseId": "dc8ec9f4f04bfae47cc257d2b40052bd", "fullName": "tests.test_loging.TestLoging#test_login_with_valid_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_loging"}, {"name": "subSuite", "value": "TestLoging"}, {"name": "host", "value": "kengni"}, {"name": "thread", "value": "3344098-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_loging"}]}